GPU/TPCFastTransformation/TPCFastSpaceChargeCorrection.h

```diff
@@ -247,6 +254,8 @@ class TPCFastSpaceChargeCorrection : public FlatObject
   RowInfo* mRowInfoPtr;           //! (transient!!) pointer to RowInfo array inside the mFlatBufferPtr buffer
   SliceRowInfo* mSliceRowInfoPtr; //! (transient!!) pointer to SliceRowInfo array inside the mFlatBufferPtr
 
+  float fInterpolationSafetyMargin{0.1}; // 10% area around the TPC row. Outside of this area the interpolation returns the boundary values.
```

Comment:
Can this be put as last data member? Also, there is no ClassDef, but we have it in the LinkDef: https://github.com/AliceO2Group/AliceO2/blob/dev/GPU/TPCFastTransformation/TPCFastTransformationLinkDef_O2.h#L67
also the SliceInfo https://github.com/AliceO2Group/AliceO2/blob/dev/GPU/TPCFastTransformation/TPCFastTransformationLinkDef_O2.h#L66
Is this intentional?